contracts/airdrops/_deprecated_airdrops/ItemNFTDrop.sol

```diff
@@ -1,9 +1,9 @@
 /* // DEPRECATED CONTRACT
 
 // SPDX-License-Identifier: UNLICENSED
-pragma solidity ^0.8.15;
+pragma solidity ^0.8.16;
 
-import "@rari-capital/solmate/src/tokens/ERC1155.sol";
```

Comment:
I'd remove _deprecated_airdrops folder. Git is a versioning system, no need to have it present.